app/assets/stylesheets/application.scss

```diff
@@ -13,9 +13,28 @@
  *= require_tree .
  *= require_self
  *= require foundation_and_overrides
-
  */
 .cancel {
   text-align: center;
+}
+
+body {
+  background-image: image_url("");
+  background-position: center top;
+  background-position: 0 45px;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.welcome {
+  background: rgba(255, 255, 255, 0.5);
+  padding: 10px;
+}
 
+.footer {
+  background-color: #333333;
+  bottom: 0;
+  color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.